### PR TITLE
Fixes set-env depreciation

### PR DESCRIPTION
**Pull request by ZeWaka**
Remake of #1129

[Why]: # (If this does not close or work on an existing GitHub issue, please add a short description [two lines down] of why you think these changes would benefit the server. If you can't justify it in words, it might not be worth adding.)

[GitHub bad.](https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/) Depreciation bad. CI breaking in the future bad. 

This fix. Gooncoder happy.

### CODE REVIEW

File: .github/workflows/ci-suite.yml [hunk 1]
```diff
@@ -186,25 +186,25 @@ jobs:
     steps:
     - name: Set General__UseBasicWatchdog
       if: ${{ matrix.watchdog-type == 'Basic' }}
-      run: echo "::set-env name=General__UseBasicWatchdog::true"
+      run: echo "General__UseBasicWatchdog=true" >> $GITHUB_ENV
 
     - name: Set TGS4_TEST_CONNECTION_STRING
-      run: echo "::set-env name=TGS4_TEST_CONNECTION_STRING::Server=(localdb)\MSSQLLocalDB;Integrated Security=true;Initial Catalog=TGS_${{ matrix.watchdog-type }}_${{ matrix.configuration }};Application Name=tgstation-server"
+      run: echo "TGS4_TEST_CONNECTION_STRING=Server=(localdb)\MSSQLLocalDB;Integrated Security=true;Initial Catalog=TGS_${{ matrix.watchdog-type }}_${{ matrix.configuration }};Application Name=tgstation-server" >> $GITHUB_ENV
```
Comment (Cyberboss) [inline .github/workflows/ci-suite.yml:192]: This change is sus methinks.

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-suite.yml (hunk 1 above)
Comment (ZeWaka) [reply .github/workflows/ci-suite.yml:192]: grr, why wasn't this using Write-Host then

**Comment by ZeWaka** [comment]
on you now

**Comment by Cyberboss** [comment]
The Windows integration test is still silently failing cause of the db connection string. https://github.com/tgstation/tgstation-server/pull/1130/checks?check_run_id=1211358049 in that line i pointed out there are a lot of `=`s and i think it's messing up the parsing

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**